libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.java

```diff
@@ -126,6 +128,22 @@ public static Location createFirstLocationFromRoute(DirectionsRoute route) {
     return forcedLocation;
   }
 
+  /**
+   * Checks if the {@link String} route profile provided is a valid profile
+   * that can be used with the directions API.
+   *
+   * @param routeProfile being validated
+   * @return true if valid, false if not
+   * @since 0.13.0
+   */
+  public static boolean isValidRouteProfile(String routeProfile) {
+    return !TextUtils.isEmpty(routeProfile)
+      && routeProfile.contentEquals(DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
```

Comment:
What about extracting the valid profiles into a collection and query only once (instead of 4 as is right now)? Does that make sense?